### ISSUE TITLE
RestQueryTester.java throws NumberFormatException in 7.0.5 JAVA / REST API Stress Test

### ISSUE THREAD

**Issue by sammefford** [marklogic/java-client-api#259]
From Ajit George:

Had a setup comprising of 1 E Node and 2 Nodes to test JAVA / REST Stress Test. The stress client was run from another server.
 
 
After starting the test, looking at the script log, it was found that RestQueryTester class throws NumberFormatException.
 
Here is the stack trace:
 
```
10:44:23 About to load 2501 documents.
10:44:23 About to load 2501 documents.
Entering int endFinishedTests()
Exiting int endFinishedTests()0
Currently running 25
MinUsers 20
10:44:51 Querying...
10:44:52 Querying...
10:44:52 Querying...
10:44:52 Querying...
Entering int endFinishedTests()
Exiting int endFinishedTests()0
Currently running 25
MinUsers 20
java.lang.NumberFormatException: null
    at java.lang.Long.parseLong(Long.java:375)
    at java.lang.Long.parseLong(Long.java:468)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.handleResponse(SearchHandle.java:1035)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.handleTop(SearchHandle.java:1016)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.parse(SearchHandle.java:984)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.access$100(SearchHandle.java:948)
    at com.marklogic.client.io.SearchHandle.receiveContent(SearchHandle.java:157)
    at com.marklogic.client.io.SearchHandle.receiveContent(SearchHandle.java:72)
    at com.marklogic.client.impl.QueryManagerImpl.search(QueryManagerImpl.java:199)
    at com.marklogic.client.impl.QueryManagerImpl.search(QueryManagerImpl.java:162)
    at test.stress.JavaAPISession.search(JavaAPISession.java:266)
    at test.stress.RestQueryTester.runTest(RestQueryTester.java:89)
    at test.stress.StressTest.run(StressTest.java:71)
java.lang.NumberFormatException: null
    at java.lang.Long.parseLong(Long.java:375)
    at java.lang.Long.parseLong(Long.java:468)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.handleResponse(SearchHandle.java:1035)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.handleTop(SearchHandle.java:1016)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.parse(SearchHandle.java:984)
    at com.marklogic.client.io.SearchHandle$SearchResponseImpl.access$100(SearchHandle.java:948)
    at com.marklogic.client.io.SearchHandle.receiveContent(SearchHandle.java:157)
10:44:52 Test took: 29314ms at com.marklogic.client.io.SearchHandle.receiveContent(SearchHandle.java:72)
    at com.marklogic.client.impl.QueryManagerImpl.search(QueryManagerImpl.java:199)
    at com.marklogic.client.impl.QueryManagerImpl.search(QueryManagerImpl.java:162)
.......
```

**Comment by sammefford** [comment]
This is because to improve Search API performance some changes from 8 were back-ported to 7 which conditionally skip estimating the total, and as part of that do not return the total attribute.  I will change the b2_0 branch like I did on the develop branch to not fail when the total attribute is missing.

**Comment by sammefford** [comment]
Fixed by commit f251e44

**Comment by georgeajit** [comment]
Run a Stress test with latest scripts and ML Server. The script logs do not have the reported NumberFormatExceptions.
